# HG changeset patch
# User Rik <rik@octave.org>
# Date 1382648025 25200
#      Thu Oct 24 13:53:45 2013 -0700
# Node ID dffb1e9a6d3b0f3d994c4232bb064d1b0796ccb5
# Parent  887df64e54e4a0c1eb40cff3b8ea4fa09073eba6
doc: Final spellcheck of documentation before 3.8 release.

* doc/interpreter/doccheck/aspell-octave.en.pws: Add new words to
Octave-specific dictionary.

* scripts/signal/stft.m: Final spellcheck of documentation before 3.8 release.

diff --git a/doc/interpreter/doccheck/aspell-octave.en.pws b/doc/interpreter/doccheck/aspell-octave.en.pws
--- a/doc/interpreter/doccheck/aspell-octave.en.pws
+++ b/doc/interpreter/doccheck/aspell-octave.en.pws
@@ -404,16 +404,17 @@ gunzipped
 gz
 gzip
 gzipped
 Hackbusch
 Hadamard
 Haddad
 HandleVisibility
 Hankel
+hanning
 Hanning
 hardcode
 hardcoding
 Hauberg
 HDF
 hdf
 headerlines
 heartsuit
@@ -424,16 +425,17 @@ Hessenberg
 heteroscedascity
 hggroup
 hggroups
 hh
 HH
 Higham
 Hindmarsh
 Hindmarsh's
+histc
 Hodel
 holomorphic
 Horner's
 horzcat
 hostname
 hotelling
 Hotelling's
 HSV
@@ -1162,25 +1164,27 @@ windowstyle
 WinRand
 WIPO
 wireframe
 wp
 wspace
 xb
 xc
 xcorr
+xdata
 xdigit
 xerrorbar
 xerrorbars
 XFig
 xGBTRF
 xGELSD
 xGETRF
 xGTSV
 xl
+xmax
 xmin
 xPBTRF
 xPOTRF
 xPTSV
 xtest
 xTRTRS
 xu
 xwd
@@ -1188,16 +1192,18 @@ xy
 xyerrorbar
 xyerrorbars
 xyz
 yc
 ydata
 yerrorbar
 yerrorbars
 ylim
+ymax
+ymin
 yy
 YY
 yyyy
 YYYY
 yyyymmdd
 yyyymmddTHHMMSS
 Zechner
 Ziggurat
diff --git a/scripts/signal/stft.m b/scripts/signal/stft.m
--- a/scripts/signal/stft.m
+++ b/scripts/signal/stft.m
@@ -38,17 +38,17 @@
 ## win_type = 2
 ##
 ## @item @qcode{"rectangle"}
 ## win_type = 3
 ## @end table
 ##
 ## The window names can be passed as strings or by the @var{win_type} number.
 ##
-## The following defaults are used for unspecifed arguments:
+## The following defaults are used for unspecified arguments:
 ## @var{win_size} = 80, @var{inc} = 24, @var{num_coef} = 64, and
 ## @var{win_type} = 1.
 ##
 ## @code{@var{y} = stft (@var{x}, @dots{})} returns the absolute values
 ## of the Fourier coefficients according to the @var{num_coef} positive
 ## frequencies.
 ##
 ## @code{[@var{y}, @var{c}] = stft (@code{x}, @dots{})} returns the
